endpoints/bus/realtime.js

```diff
@@ -3,12 +3,26 @@
 const h = require('apis-helpers')
 const request = require('request')
 const app = require('../../server')
-const isn2wgs = require('isn2wgs')
 
 const debug = require('debug')('bus/realtime')
 
 const getBusRoutes = (data) => new Promise((resolve, reject) => {
-  request.get('http://straeto.is/bitar/bus/livemap/json.jsp', function (error, response, body) {
+  const reykjavikBusRoutes = [1, 2, 3, 4, 5, 6, 11, 12, 13, 14, 15, 16, 17, 18, 21, 22, 23, 24, 26, 27, 28, 29, 31, 33, 34, 35, 43, 44],
+    southBusRoutes = [51, 52, 71, 72, 73, 75],
+    northWestBusRoutes = [57, 58, 59, 81, 82, 83, 84, 85, 86],
+    northEastBusRoutes = [56, 78, 79],
+    akureyriBusRoutes = ['A1', 'A2', 'A3', 'A4', 'A5', 'A6'],
+    sudurnesBusRoutes = [55, 87, 88, 88, 89],
+    reykjanesBusRoutes = ['R1', 'R2', 'R3', 'R4']
+  const allBusRoutes = reykjavikBusRoutes.concat(
+    southBusRoutes,
+    northWestBusRoutes,
+    northEastBusRoutes,
+    akureyriBusRoutes,
+    sudurnesBusRoutes,
+    reykjanesBusRoutes)
+
+  request.get('https://app.straeto.is/pele/api/v1/positions/filter/' + allBusRoutes.join('%2C'), function (error, response, body) {
```

Comment:
Agree, also the new endpoint can handle invalid route numbers so there's no point in doing this check.